src/screens/CreatePost/index.js

```diff
@@ -60,7 +62,6 @@ import {
 } from '../../utils/constants';
 import {PROFILE_CACHE} from '../../utils/cache/constant';
 import {ShowingAudience, createPost} from '../../service/post';
-import {composeImageMeta} from '../../utils/string/file';
 import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {
   getDurationId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `composeImageMeta` import is fine as long as it's not used anywhere else in the file. If it's still being used, this would cause a reference error.



<!-- This is an auto-generated comment by OSS CodeRabbit -->